### ISSUE TITLE
Multiple release and branch information

### ISSUE THREAD

**Issue by rajmohanb** [sipwise/rtpengine#901]
Multiple versions are released in parallel such as 6.x, 7.x and 8.x series. I did not find any information here on what are the differences between these various parallel tracks. For anyone new and using a particular release, any documentation would help to follow along a particular release or switch to another one.
Hence request you to add some documentation on various parallel tracks and what makes them different.

**Comment by taurus-forever** [comment]
Hi,

rtpengine is a part of Sipwise NCGP platform and follows Sipwise releasing policy.

TL;DR:
all new features are going to trunk/new release only (mr8.2 at the moment).
Here you might notice regression on upgrade.
All bugfixes are fixed in trunk and backported for all the supported releases.
Sipwise NGCP publishes new release each 8 weeks, list of supported releases you can find here: https://www.sipwise.com/releases/
Also Sipwise provides support for LTS (long-time supported) releases for 3 years.
This is why you see so many versions here.

The best way to see changes is to execute "git log <source_release>..<target_release>":
```
15:14:20 ✔ taurus:(master)~/sipwise/git/rtpengine$ git log --oneline mr6.5.5..mr6.5.6
a034701 (tag: mr6.5.6.2, origin/mr6.5.6, mr6.5.6) Release new version 6.5.6.2+0~mr6.5.6.2
e1b3a39 TT#64601 refuse overriding crypto params from SDES when DTLS is in use
159a06b (tag: mr6.5.6.1) Release new version 6.5.6.1+0~mr6.5.6.1
54bf5ab don't skip address family selection when zero port is present
c7c30df TT#65800 fix SDES negotiation bug
b87b08e TT#65100 don't start a redis notification context/thread when not in use
b34553c TT#65100 properly cleanup redis async context
f429782 TT#65100 Fix double free for redis context on closure
1530141 TT#65000 fix kernel module compilation for 5.2 kernels
e722a0a TT#64301 log xmlrpc children to stderr
4c0ba30 TT#64050 add `G729a` as an incorrect alternative to `G729`
d1bd538 TT#63901 don't free redis event base while being used
7e5b50e TT#63550 fix mem allocation bug resulting in mem corruption during redis restore
4bbc261 Release new version 6.5.6.0+0~mr6.5.6.0
15:14:25 ✔ taurus:(master)~/sipwise/git/rtpengine$
```

Also you can find Debian changelog committed to every branch:
https://github.com/sipwise/rtpengine/blob/master/debian/changelog

Feel free to ask your questions if you need more details here.
Thank you!

**Comment by nicoonline** [comment]
Hi,

What is the difference between 6.5.5 and 6.5.6 or 6.5.7 ?

**Comment by taurus-forever** [comment]
TL;DR: bugfixes only.

mr6.5.**5** is a "fifth build" (aka service pack in Windows) of LTS release mr6.5
We stopped developing mr6.5.**5** because new "build" mr6.5.**6** is available.
The latest build (mr6.5.7) provides all bugfixes we currently have addressed.

New builds will be released accordingly to the calendar: https://www.sipwise.com/releases/
Feel free to ask more questions here it if is still not clear for you!
